package.json

```
@@ -27,7 +27,8 @@
     "lint:md": "remark . --quiet --frail",
     "prepare": "husky",
     "test": "npm run lint",
```

[suggest] Maybe this is the last suggestion. How about adding a test executed only on CI? It's possible to extend the test script in the future.

```suggestion
    "pretest": "npm run lint",
    "test": "node scripts/test.js",
```

`scripts/test.js`:

```js
import { execSync } from 'node:child_process';
import process from 'node:process';

if (process.env.CI) {
	execSync('npm run update-readme', { stdio: 'inherit' });
	execSync('npm run update-workflows', { stdio: 'inherit' });
	execSync('git diff --exit-code README.md .github/workflows', { stdio: 'inherit' });
}
```